packages/twenty-front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -250,13 +252,37 @@ export const CommandMenu = () => {
     callback: closeCommandMenu,
   });
 
-  const selectableItemIds = matchingCreateCommand
-    .map((cmd) => cmd.id)
+  const isAskAIEnabled = useIsFeatureEnabled('IS_ASK_AI_ENABLED');
+
+  const askAiCommand: Command = {
+    id: 'ask-ai',
+    to: '', // TODO
+    Icon: IconSparkles,
+    label: 'Ask AI',
+    type: CommandType.Navigate,
+    onCommandClick: async () => {
+      const queryResult = await getTextToSql({
+        variables: { text: commandMenuSearch },
+      });
+      console.log('queryResult', queryResult);
+      alert(JSON.stringify(queryResult.data?.getTextToSQL, undefined, 2));
+      // TODO: Call openActivityRightDrawer
+    },
+  };
+
+  const askAiCommandIds: string[] = isAskAIEnabled ? [askAiCommand.id] : [];
+
+  const selectableItemIds = askAiCommandIds
+    .concat(matchingCreateCommand.map((cmd) => cmd.id))
     .concat(matchingNavigateCommand.map((cmd) => cmd.id))
     .concat(people.map((person) => person.id))
     .concat(companies.map((company) => company.id))
     .concat(activities.map((activity) => activity.id));
 
+  const [getTextToSql, { data, loading, error }] = useGetTextToSqlLazyQuery({
+    variables: { text: 'How many employees does Stripe have?' },
```

Comment:
is this the default value if nothing is asked? I could look more but at first glance I don't really understand why we have that query